live-examples/js-examples/math-asinh.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">console.log(Math.acosh(0.999999999999));
+// expected output: NaN
+
+console.log(Math.acosh(1));
+// expected output: 0
+
+console.log(Math.acosh(2));
+// expected output: 1.3169578969248166
+
+console.log(Math.acosh(2.5));
```

Comment:
This is the wrong example

Reply:
oops sorry - not sure how that slipped thru :/

Reply:
I think this one still needs to be done, or am I confused?